xbmc/cores/RetroPlayer/cheevos/Cheevos.h

```diff
@@ -12,6 +12,9 @@
 
 #include <map>
 #include <string>
+#include <vector>
+#include<unordered_map>
+using std::vector;
```

Comment:
try to avoid importing unrelated namespaces outside of the ones this class is declared within